extensions/amp-story/1.0/amp-story-share.js

```diff
@@ -117,13 +117,20 @@ function buildLinkShareItemTemplate(el) {
  */
 function buildProviderParams(opt_params) {
   const attrs = dict();
-
+  
   if (opt_params) {
     Object.keys(opt_params).forEach((field) => {
       if (field === 'provider') {
         return;
       }
-      attrs[`data-param-${field}`] = opt_params[field];
+
+      /**
+       * based on amp-social-share accepted attributes
+       * https://amp.dev/documentation/components/amp-social-share/#attributes
+       */
+      if (['data-target', 'data-share-endpoint', 'aria-label'].includes(field) || /^data-param-/.test(field)) {
+        attrs[field] = opt_params[field];
+      }
```

Comment:
This statement looks really good, but we still need to keep the old behavior for backwards compatibility.

Can you add a `else` statement and keep the old `attrs[`data-param-${field}`] = opt_params[field];` line?
If someone configured `{foo: "bar"}` is used to give `data-param-foo="bar"`. With your new code they should configure `{data-param-foo: "bar"}` but we need to be backwards compatible.

Reply:
this all looks good to me now! Lmk when you fixed the localization warning

Reply:
it's fixed on my last commit. please recheck @gmajoulet